exercises/concept/tim-from-marketing-2/src/test/java/EmployeeServiceTest.java

```diff
@@ -0,0 +1,84 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Optional;
+
+public class EmployeeServiceTest {
+
+    private EmployeeService employeeService;
+    private List<Employee> listOfEmployees = new ArrayList<>();
+
+    void initList() {
+        listOfEmployees.add(new Employee(1, "Tim", "Marketing"));
+        listOfEmployees.add(new Employee(2, "Joe", "Sales"));
+        listOfEmployees.add(new Employee(3, "Jane", "IT"));
+        listOfEmployees.add(new Employee(4, null, null));
```

Comment:
I think there are a number of cases missing,  such as a `null` name and non-null department and non-null name and null department.

Reply:
WIP